src/game_entities/movable.py

```diff
@@ -455,7 +455,7 @@ def move(self) -> None:
         Decrease the current timer according to move speed and move the entity by one tile if it's time for.
         Change the state of the entity if the movement is finished.
         """
-        self._timer -= Movable.move_speed
+        self._timer -= int(options_manager.get_option("move_speed"))
```

Comment:
I know it's just a getter method accessing an element of a dict by key, but I don't really like the fact the method would be called **every time** an entity is moving, while the value is going to be static for a given game session.

Could we cache it somehow as a runtime const like it used to be?